### PR TITLE
feat: footer v2 

### PR DESCRIPTION
**Pull request by nata-kab**
## Jira issue

JIRA issue: [FSU-245](https://klau.atlassian.net/browse/FSU-245)

## Describe your change

Footer update to v2

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/design-system/blocks/footer/Footer.styled.js [hunk 1]
```diff
@@ -102,18 +102,20 @@ export const Section = styled.div`
   gap: 20px;
 `
 
+// export const Icons = styled.div`
```
Comment (im-klau) [inline packages/design-system/blocks/footer/Footer.styled.js:105]: could you please remove all comments? 🙏

Comment (nata-kab) [reply packages/design-system/blocks/footer/Footer.styled.js:105]: 🙈

Comment (nata-kab) [reply packages/design-system/blocks/footer/Footer.styled.js]: Done :)

**Review by AmandaKr — COMMENTED** [review]
image
I see that text "Skopiowano" is not aligned with the icon 🤔

**Review by im-klau — APPROVED** [review]
Welcome back Natalia! 😄 
Looks nice!

**Review by AmandaKr — APPROVED** [review]

**Pull request merged**